docs/en/setup/backend/backend-fetcher.md

```diff
@@ -0,0 +1,13 @@
+# Open Fetcher
+Fetcher is a concept in SkyWalking backend. It uses pulling mode rather than [receiver](backend-receivers.md), which
+read the data from the target systems. This mode is typically in some metrics SDKs, such as Prometheus.
+
+## Prometheus Fetcher
+```yaml
+prometheus-fetcher:
+  selector: ${SW_PROMETHEUS_FETCHER:default}
```

Comment:
BTW, I'm a little confused about the "selectors", which scenariodos do we need to provide our own Prometheus fetcher provider? 🤔️

Reply:
What do you mean? I think in any case, we need our own fecthcer, otherwise, who will fetch the data?